lxc/list.go

```diff
@@ -621,10 +621,10 @@ func (c *cmdList) IP6ColumnData(cInfo api.InstanceFull) string {
 
 func (c *cmdList) typeColumnData(cInfo api.InstanceFull) string {
 	if cInfo.Ephemeral {
-		return i18n.G("EPHEMERAL")
+		return fmt.Sprintf("%s (EPHEMERAL)", i18n.G(strings.ToUpper(cInfo.Type)))
 	}
 
-	return i18n.G("PERSISTENT")
+	return fmt.Sprintf("%s", i18n.G(strings.ToUpper(cInfo.Type)))
```

Comment:
Should just be `fmt.Sprintf("%s", strings.ToUpper(cInfo.Type))`

Reply:
Well, actually, just `return strings.ToUpper(cInfo.Type)`

Reply:
@stgraber do we not want to translate the instance type?